{{cookiecutter.project_slug}}/{{cookiecutter.project_slug}}/config.py

```diff
@@ -64,27 +75,23 @@ class DevConfig(BaseConfig):
     AUTO_UPGRADE = True
     {% endif %}
```

Comment:
I think this {% endif %} should have been deleted, no?

Reply:
It collapsed the rest of the code. Expand it and you should be able see the `{% if`

Reply:
It collapsed the rest of the code. Expand it and you should be able see the {% if

Reply:
Derp. My bad.